source/docs/api-reference/api-usage/status-signals.rst

```diff
@@ -134,13 +134,14 @@ Timestamps
 ^^^^^^^^^^
 
 The timestamps of a ``StatusSignalValue`` can be retrieved by calling ``getAllTimestamps()``, which returns a collection of ``Timestamp`` (`Java <https://api.ctr-electronics.com/phoenixpro/release/java/com/ctre/phoenixpro/Timestamp.html>`__, `C++ <https://api.ctr-electronics.com/phoenixpro/release/cpp/classctre_1_1phoenixpro_1_1_timestamp.html>`__) objects.
+
 The ``Timestamp`` objects can be used to perform latency compensation math.
```

Comment:
This was meant to be one paragraph, so combine the lines instead of adding another newline.